File: Scripts/newTask/exponentiation-operator/javascript/exponentiation-operator.js
function pow(base, exp) {
    if (exp != Math.floor(exp))
        throw "exponent must be an integer";
    if (exp < 0)
        return 1 / pow(base, -exp);
    var ans = 1;
    while (exp > 0) {
        ans *= base;
        exp--;
    }
    return ans;
}

for ( var i = 0; i < 1000000000; ++i) {
pow(2017,12);
pow(19.88,12);
}

File: Scripts/newTask/exponentiation-operator/ruby/exponentiation-operator.rb
class Numeric
  def pow(m)
    raise TypeError, "exponent must be an integer: #{m}" unless m.is_a? Integer
    Array.new(m, self).reduce(1, :*)
  end
end

  for i in 0..1000000000 
	2017.pow(12)
	19.88.pow(12)
  end

